core/src/main/java/org/frankframework/receivers/Receiver.java

```diff
@@ -2036,7 +2033,7 @@ public void setMessageLog(ITransactionalStorage<Serializable> messageLog) {
 
 	/**
 	 * Sets the name of the Receiver, as known to the Adapter.
-	 * If the listener implements the {@link INamedObject name} interface and <code>getName()</code>
+	 * If the listener implements the {@link NameAware name} interface and <code>getName()</code>
```

Comment:
```suggestion
	 * If the <code>getName()</code>
```

Listener always implements `NameAware` and `HasName` since `IListener` extends these interfaces.